net.wz2100.wz2100.yaml

```diff
@@ -44,16 +44,33 @@ modules:
     sources:
       - type: git
         url: https://github.com/fribidi/fribidi
-        tag: v1.0.8
-        commit: 2e9bd91cfd857c5b78a210d8f24862a4bb704059
+        tag: v1.0.9
+        commit: f9e8e71a6fbf4a4619481284c9f484d10e559995
       - type: patch
         path: disable-docs.patch
-      
+
+  - name: libsodium
+    sources:
+      - type: archive
+        url: https://download.libsodium.org/libsodium/releases/libsodium-1.0.18.tar.gz
+        sha256: 6f504490b342a4f8a4c4a02fc9b866cbef8622d5df4e5452b46be121e46636c1
+
+  - name: miniupnpc
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        sha256: e19fb5e01ea5a707e2a8cb96f537fbd9f3a913d53d804a3265e3aeab3d2064c6
+        url: https://miniupnp.tuxfamily.org/files/miniupnpc-2.1.tar.gz
+
   - name: wz2100
+    buildsystem: "cmake"
+    builddir: true
+    config-opts:
+      - -DENABLE_DOCS=OFF
```

Comment:
There is a WZ_DISTRIBUTOR variable available; should I set it to `Flatpak`?

Reply:
👍 Probably should be set to `Flathub`.